packages/expo-updates/e2e/__tests__/setup/create-updates-test.js

```diff
@@ -8,7 +8,7 @@ const workingDir = path.resolve(repoRoot, '..');
 /*
  * Change this to your own Expo account name
  */
-const EXPO_ACCOUNT_NAME = 'myusername';
+const EXPO_ACCOUNT_NAME = process.env.EXPO_ACCOUNT_NAME || 'myusername';
```

Comment:
revert?

Reply:
This script is mainly for use by Expo developers for internal testing of this flow and other manual update tests. I do want this change, but I can put it in a different PR.